### PR TITLE
feat(editor/subscriber): Subscribe and unsubscribe for Subscriber Logical Nodes (GOOSE/SMV)

### LINKED ISSUE #933: Subscriber - Logical Node (SMV): Disconnect FCDA from ExtRef
As a user of OpenSCD I want to disconnect an existing SMV subscriber connection

**Requirements:**
- Remove element `ExtRef` that is connected to the selected `FCDA`
- Make sure that the `Inputs` is no leaf element (schema invalid)

### LINKED ISSUE #934: Subscriber - Logical Node (SMV): Connect FCDA to ExtRef
As a user of OpenSCD I want to connect a `FCDA` element to the `ExtRef` element. 

**Requirements**
- create new `ExtRef` for a non-later binding type connection
- add Ed1 attributes
1. copy`ldInst`, `prefix`, `lnClass`, `lnInst`, `doName` and  `daName` to the `ExtRef` element
2. add `iedName` that is the `name` attribute of the `IED` element `FCDA` resides in
- add Ed2 attributes (>version="2003")
1. `serviceType` is `SMV`
2. `srcLDInst` is the `inst` attribute of the `LDevice` element where the source `SampledValueControl` resides in
5. `srcPrefix` is empty string as `SampledValueControl` are always in `LN0`
6. `srcLNClass` is `LLN0` as `SampledValueControl` are always in `LN0`
7. `srcLNInst` is null as `SampledValueControl` are always in `LN0` (empty string is schema invalid)
8. `srcCBName` is the `name` attribute of the `SampledValueControl` 
- add the `ExtRef` to the `Inputs` within the logical node `LN` or `LN0` selected from the right-hand side list
- make this a default plugin: set default setting in plugin registry to true.

### LINKED ISSUE #938: Subscriber - Logical Node (GOOSE): Disconnect FCDA from ExtRef
As a user of OpenSCD I want to disconnect an existing GOOSE subscriber connection

**Requirements:**
- Remove element `ExtRef` that is connected to the selected `FCDA`
- Make sure to also remove the `Inputs` when the last 'ExtRef' is removed (schema invalid)

### LINKED ISSUE #939: Subscriber - Logical Node (GOOSE): Connect FCDA to ExtRef
As a user of OpenSCD I want to connect a `FCDA` element to the `ExtRef` element. 

**Requirements**
- create new `ExtRef` for a non-later binding type connection
- add Ed1 attributes
1. copy`ldInst`, `prefix`, `lnClass`, `lnInst`, `doName` and  `daName` to the `ExtRef` element
2. add `iedName` that is the `name` attribute of the `IED` element `FCDA` resides in
- add Ed2 attributes
1. `serviceType` is `GOOSE`
2. `srcLDInst` is the `inst` attribute of the `LDevice` element where the source `GSEControl` resides in
5. `srcPrefix` is empty string as `GSEControl` are always in `LN0`
6. `srcLNClass` is `LLN0` as `GSEControl` are always in `LN0`
7. `srcLNInst` is null as `GSEControl` are always in `LN0` (empty string is schema invalid)
8. `srcCBName` is the `name` attribute of the `GSEControl` 
- add the `ExtRef` to the `Inputs` within the logical node `LN` or `LN0` selected from the right-hand side list
- make this a default plugin: set default setting in plugin registry to true.

### PR DESCRIPTION
**Pull request by dlabordus**
closes #939 
closes #938 
closes #934
closes #933

### CODE REVIEW

File: src/editors/subscription/later-binding/ext-ref-ln-binding-list.ts [hunk 1]
```diff
@@ -106,6 +124,68 @@ export class ExtRefLnBindingList extends LitElement {
...
+      )
+    ) {
+      const extRef = createExtRefElement(
+        this.currentSelectedControlElement,
+        this.currentSelectedFcdaElement
+      );
+      actions.push({ new: { parent: inputsElement, element: extRef } });
+    }
+
+    const title = 'Connect';
```
Comment (JakobVogelsang) [inline src/editors/subscription/later-binding/ext-ref-ln-binding-list.ts:157]: A translation here would be nice. I don't know whether is missing in the subscriber message binding as well:

en: Connect data attribute
de: Daten-Attribut verbunden

File: src/editors/subscription/later-binding/ext-ref-ln-binding-list.ts [hunk 2]
```diff
@@ -106,6 +124,68 @@ export class ExtRefLnBindingList extends LitElement {
...
+    );
+    if (extRefElement) {
+      actions.push({ old: { parent: inputElement, element: extRefElement } });
+    }
+
+    // Check if empty Input Element should also be removed.
+    actions.push(...emptyInputsDeleteActions(actions));
+
+    return {
+      title: 'Disconnect',
```
Comment (JakobVogelsang) [inline src/editors/subscription/later-binding/ext-ref-ln-binding-list.ts:184]: Same here:

en: Disconnect data attribute
de: Daten-Attribute Verbindung gelöst

File: test/integration/editors/GooseSubscriberDataBinding.test.ts [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+    element = await fixture(
+      html` <goose-subscriber-data-binding-plugin
+        .doc="${doc}"
+        .nsdoc=${nsdoc}
+      ></goose-subscriber-data-binding-plugin>`
+    );
+  });
+
+  it('when subscribing an available ExtRef then the lists are changed', async () => {
```
Comment (JakobVogelsang) [inline test/integration/editors/GooseSubscriberDataBinding.test.ts:37]: I would love to see in a separate test that the `ExtRef` has been also added to the XMLDocument. Another test that might be critical is the removing of the last `ExtRef` in an input results in removing the `Input` element too

**Review by JakobVogelsang — COMMENTED** [review]
The integration tests could check for the actual manipulation of the `ExtRef` element as commented. As it is not critical, it is up to you.

File: src/editors/subscription/later-binding/ext-ref-ln-binding-list.ts (hunk 2 above)
Comment (dlabordus) [reply src/editors/subscription/later-binding/ext-ref-ln-binding-list.ts:184]: Fixed also for message binding. Juan and I already noticed this.
Maybe in the future create a more descriptive message, that includes the SMV/GOOSE Name or info about the FCDA.

**Review by JakobVogelsang — APPROVED** [review]
LGTM

**Pull request merged**